File: CS/WinWebSolution.Module/Updater.cs
using System;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;

namespace WinWebSolution.Module {
    public class Updater : ModuleUpdater {
        public Updater(IObjectSpace os, Version currentDBVersion) : base(os, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            Order order = ObjectSpace.CreateObject <Order>();
            order.Name = "Order 1";
            for (int i = 0; i < 5; i++) {
                OrderItem item = ObjectSpace.CreateObject<OrderItem>();
                item.Subject = "OrderItem " + i.ToString();
                item.Order = order;
                item.UpdateModifiedOn(DateTime.Now + TimeSpan.FromDays(i));
                item.Save();
            }
        }
    }
}

File: VB/WinWebSolution.Module/Updater.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating

Namespace WinWebSolution.Module
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal os As IObjectSpace, ByVal currentDBVersion As Version)
			MyBase.New(os, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			Dim order As Order = ObjectSpace.CreateObject (Of Order)()
			order.Name = "Order 1"
			For i As Integer = 0 To 4
				Dim item As OrderItem = ObjectSpace.CreateObject(Of OrderItem)()
				item.Subject = "OrderItem " & i.ToString()
				item.Order = order
				item.UpdateModifiedOn(DateTime.Now + TimeSpan.FromDays(i))
				item.Save()
			Next i
		End Sub
	End Class
End Namespace
